test/KeyEventsUnits.ipynb

```diff
@@ -0,0 +1,1088 @@
+       ".output_html .mf { color: #666666 } /* Literal.Number.Float */\n",
+       ".output_html .mh { color: #666666 } /* Literal.Number.Hex */\n",
+       ".output_html .mi { color: #666666 } /* Literal.Number.Integer */\n",
+       ".output_html .mo { color: #666666 } /* Literal.Number.Oct */\n",
+       ".output_html .sa { color: #BA2121 } /* Literal.String.Affix */\n",
+       ".output_html .sb { color: #BA2121 } /* Literal.String.Backtick */\n",
+       ".output_html .sc { color: #BA2121 } /* Literal.String.Char */\n",
+       ".output_html .dl { color: #BA2121 } /* Literal.String.Delimiter */\n",
+       ".output_html .sd { color: #BA2121; font-style: italic } /* Literal.String.Doc */\n",
+       ".output_html .s2 { color: #BA2121 } /* Literal.String.Double */\n",
+       ".output_html .se { color: #BB6622; font-weight: bold } /* Literal.String.Escape */\n",
+       ".output_html .sh { color: #BA2121 } /* Literal.String.Heredoc */\n",
+       ".output_html .si { color: #BB6688; font-weight: bold } /* Literal.String.Interpol */\n",
+       ".output_html .sx { color: #008000 } /* Literal.String.Other */\n",
+       ".output_html .sr { color: #BB6688 } /* Literal.String.Regex */\n",
+       ".output_html .s1 { color: #BA2121 } /* Literal.String.Single */\n",
+       ".output_html .ss { color: #19177C } /* Literal.String.Symbol */\n",
+       ".output_html .bp { color: #008000 } /* Name.Builtin.Pseudo */\n",
+       ".output_html .fm { color: #0000FF } /* Name.Function.Magic */\n",
+       ".output_html .vc { color: #19177C } /* Name.Variable.Class */\n",
+       ".output_html .vg { color: #19177C } /* Name.Variable.Global */\n",
+       ".output_html .vi { color: #19177C } /* Name.Variable.Instance */\n",
+       ".output_html .vm { color: #19177C } /* Name.Variable.Magic */\n",
+       ".output_html .il { color: #666666 } /* Literal.Number.Integer.Long */</style><div class=\"highlight\"><pre><span></span>\n",
+       "</pre></div>\n"
+      ],
+      "text/latex": [
+       "\\begin{Verbatim}[commandchars=\\\\\\{\\}]\n",
+       "\n",
+       "\\end{Verbatim}\n"
+      ],
+      "text/plain": []
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "%%ignite\n",
+    "# Unit test: key_pressed, key_repeated, special keys\n",
+    "def setup():\n",
+    "    size(500, 500)\n",
+    "\n",
+    "repeated_key = \"\"\n",
+    "repeated_count = 0\n",
+    "    \n",
+    "def draw():\n",
+    "    background(\"default\")\n",
+    "    text_size(50)\n",
+    "    text(\"Repeated key\", 50, 150)\n",
+    "    text(repeated_key, 50, 200)\n",
+    "    text(f\"{repeated_count} times\", 50, 250)\n",
+    "\n",
+    "def key_pressed():\n",
+    "    global repeated_key, repeated_count\n",
+    "    repeated_key = key\n",
+    "    repeated_count = 0\n",
+    "    \n",
+    "def key_repeated():\n",
+    "    global repeated_key, repeated_count\n",
+    "    repeated_key = key\n",
```

Comment:
Small detail - your test is missing the `<space>` output

Reply:
Fixed, and reworked the other examples for more efficiency in the event handler